curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/663353465bfb14259717da93.md

```diff
@@ -0,0 +1,388 @@
+---
+id: 663353465bfb14259717da93
+title: Step 12
+challengeType: 20
+dashedName: step-12
+---
+
+# --description--
+
```

Comment:
It's not easy to get the correct output. Maybe we need to recall how to format strings.